src/sdgio-edit.owl

```diff
@@ -3291,59 +3292,48 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
-ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020237> <http://purl.unep.org/sdg/SDGIO_00020299>)
-
-# Individual: <http://purl.unep.org/sdg/SDGIO_00020302> (adaptation to and mitigation of climate change through agricultural research)
-
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020302> "A cross-cutting objective which specifies the objective of the planned processes undertaken by the CGIAR such that they improve adaptation to and mitigation of climate change through agricultural research.")
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020302> <http://orcid.org/0000-0062-3253-4387>)
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020302> "2019-05-08T21:18:39Z"^^xsd:dateTime)
-AnnotationAssertion(rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020302> "All research and development activities need to build in resilience to climate shocks and a focus on adaptation to and mitigation of climate change. Agriculture including forestry could be the most efficient contributor to the reduction of greenhouse gases through storage in the biomass and the healthy soils. CGIAR is committed to devoting
-at least 60% of its research to those issues.")
-AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020302> "adaptation to and mitigation of climate change through agricultural research"@en)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020302> "A cross-cutting objective which specifies the objective of the planned processes undertaken by the CGIAR such that they improve adaptation to and mitigation of climate change through agricultural research. As CIGIAR notes: \"All research and development activities need to build in resilience to climate shocks and a focus on adaptation to and mitigation of climate change. Agriculture including forestry could be the most efficient contributor to the reduction of greenhouse gases through storage in the biomass and the healthy soils. CGIAR is committed to devoting
+at least 60% of its research to those issues.\"")
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020302> "adaptat to and mitigate climate change through agricultural research"@en)
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020300> <http://purl.unep.org/sdg/SDGIO_00020302>)
 
-# Individual: <http://purl.unep.org/sdg/SDGIO_00020303> (achievement of gender equity and youth employment)
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020303> (achieve gender equity and youth employment)
```

Comment:
Where? In agricultural communities? Everywhere?

Reply:
This is complicated. Maybe this CCO is about agriculture only. However, then it becomes impossible to distinguish this CCO from the CC IDO called "gender and equity achieved" (that is related to this CCO). This problem happens for every CCO and its respective CC IDO. I suggest keeping things as they are for now and open an issue about it. I will clarify either with the MARLO team (this week) or with Elizabeth when she will be back (next week).

Reply:
At the cost of introducing new issues, I make this and other CCOs more specifically related to agriculture. So, this one becomes "achieve gender equity and youth employment in the agri-food supply chains". This is imperfect though: gender equity is taken also as a goal to achieve within CGIAR, i.e. they think that now CGIAR disfavors women and they want to change. It is hard to say so much in a label though.

Reply:
I think that helps for now.